lib/matplotlib/backends/_backend_tk.py

```diff
@@ -963,10 +963,19 @@ def destroy(self, *args, **kwargs):
         self.window = None
 
 
+class HelpTk(backend_tools.ToolHelpBase):
+    def trigger(self, *args):
+        from tkinter.simpledialog import SimpleDialog
```

Comment:
Do the import at the beginning of the file?

Reply:
fixed